nuxeo-drive-core/src/main/resources/web/nuxeo.war/drive_browser_login.jsp

```diff
@@ -0,0 +1,62 @@
+<%@ page language="java"%>
+<%@ page import="org.nuxeo.runtime.api.Framework"%>
+<%@ page import="org.apache.http.HttpStatus"%>
+<%@ page import="org.nuxeo.ecm.tokenauth.service.TokenAuthenticationService"%>
+<%
+response.setCharacterEncoding("UTF-8");
+TokenAuthenticationService tokenAuthService = Framework.getService(TokenAuthenticationService.class);
+String token = tokenAuthService.acquireToken(request);
+if (token == null) {
+    response.sendError(HttpStatus.SC_UNAUTHORIZED);
+    return;
+}
+String userName = request.getUserPrincipal().getName();
+String updateToken = request.getParameter("updateToken");
+String useProtocol = request.getParameter("useProtocol");
+
+// New login system
+// If the useProtocol parameter is true, this page is opened in the user's browser
+// and the token is passed back to the application with its custom nxdrive protocol URL.
+// If useProtocol is false (for development purposes),this page is opened using
+// WebKit and the resulting JSON is parsed by Drive.
+if (Boolean.parseBoolean(useProtocol)) {
+  response.setContentType("text/html");
+%>
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<html>
+  <head>
+    <title>Nuxeo Drive startup page</title>
```

Comment:
You could use a message bundle for internationalization as in [login.jsp](https://github.com/nuxeo/nuxeo/blob/master/nuxeo-services/nuxeo-platform-web-common/src/main/resources/web/nuxeo.war/login.jsp).

So you'll need to:
- Add the `<%@ taglib uri="http://java.sun.com/jsp/jstl/fmt" prefix="fmt" %>` directive.
- Set the bundle with `<fmt:setBundle basename="messages" var="messages"/>` just before the HTML content.
- Configure the `deployment-fragment.xml` to append the messages as in nuxeo-drive-jsf.
- Define the messages in a `messages_en_US.properties` file as in nuxeo-drive-jsf.
- Use `<fmt:message bundle="${messages}" key="my.key" />` instead of hardcoded messages.